lib/routes/jiqizhixin/index.ts

```diff
@@ -0,0 +1,46 @@
+import ofetch from '@/utils/ofetch';
+import { load } from 'cheerio';
+import { parseDate } from '@/utils/parse-date';
+import { Route, DataItem } from '@/types';
+
+export const route: Route = {
+    path: '/index',
+    name: '机器之心首页',
+    url: 'jiqizhixin.com',
+    maintainers: ['ovo-tim'],
+    example: '/jiqizhixin/index',
+    description: '通过请求 HTML 获取机器之心首页，可获取10条',
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
+    handler: async (_) => {
+        const baseUrl = 'https://www.jiqizhixin.com';
+        const response = await ofetch(baseUrl, {
+            headers: {
+                'User-Agent': 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/58.0.3029.110 Safari/537.3',
+            },
+        });
+
+        const $ = load(response);
+
+        const items: DataItem[] = $('article')
+            .map((_index, article) => {
+                const titleTag = $(article).find('a.article-item__title');
+                const summaryTag = $(article).find('p.article-item__summary');
+                const dateTag = $(article).find('time');
+
+                return {
+                    title: titleTag.text().trim(),
+                    link: titleTag.attr('href') ? baseUrl + titleTag.attr('href') : undefined,
+                    description: summaryTag.text().trim(),
+                    pubDate: parseDate(dateTag.attr('datetime')!),
+                };
+            })
+            .toArray();
```

Comment:
Please use the GraphQL API instead.

Reply:
Thanks for review. But I have checked again and found that there is no specific API that can be used to capture the information.

Reply:
<details>
<summary>Screenshot</summary>

![Screenshot](https://github.com/user-attachments/assets/54be2504-a5d1-4d35-b155-7c40d29b3d18)


</details>

Reply:
That's strange, I got nothing here.
![image](https://github.com/user-attachments/assets/2a9f8f90-3864-4a78-80a7-784e38beba22)

Reply:
Try scrolling down the page

Reply:
Oh, Found that.

Reply:
The API continues to return an error code 503 even after I have obtained the CSRF token. It appears that we might need to consider using Puppeteer.

Reply:
I still can't make it work, it just keeps returning 503. :disappointed: